resources/views/waitingrooms/partials/_form_split.blade.php

```diff
@@ -6,17 +6,17 @@
     <div class="container__block">
         <div class="form-item -padded">
             <label class="field-label" for="competition_end_date">End date for these competitions:</label>
-            <input type="date" name="competition_end_date" id="competition_end_date" value='MM/DD/YYYY' class="text-field"></input>
+            <input type="date" name="competition_end_date" id="competition_end_date" class="text-field" value="{{ format_date_form_field($room, 'competition_end_date') }}"></input>
         </div>
 
         <div class="form-item -padded">
             <label class="field-label" for="id">Max users per competition:</label>
-            <input type="number" name="competition_max" id="competition_max" class="text-field" value="300" data-total="{{count($room->users)}}" }/>
+            <input type="number" name="competition_max" id="competition_max" class="text-field" data-total="{{count($room->users)}}" placeholder=300 value="{{ old('competition_max') }}" />
```

Comment:
Which is fine. They can still edit it. But it's otherwise weird that it could be left blank (cause your function that this submits to defaults to 300 anyhoo), but makes it less straightforward that it's a number that is necessary. Maybe we  keep the placeholder, but add a validation to that field to indicate it's required, apart from just being a number...?